File: apiclients/typescript/usageExampleBasic.ts
import AwaberryApiRequestClient from './awaberryapirequestclient.ts';

const client = new AwaberryApiRequestClient(
  'https://agentic.awaberry.net/apirequests',
  'yourProjectKey',
  'yourProjectSecret'
);

(async () => {
  await client.initSession();
  // Use other methods as needed
})();

File: apiclients/python/usageExampleBasic.py
from awaberryapirequestclient import AwaberryApiRequestClient

def main():
    client = AwaberryApiRequestClient(
        base_url='https://agentic.awaberry.net/apirequests',
        project_key='yourProjectKey',
        project_secret='yourProjectSecret'
    )
    try:
        client.init_session()
        # Use other methods as needed
    except Exception as e:
        print(f"An error occurred: {e}")

if __name__ == "__main__":
    main()